incompatibility/incomp-scanning.go

```diff
@@ -67,6 +84,29 @@ func writeProtobufMessage(incompatibilityReport *IncompatibilityReport, env *plu
 	env.Response.Files = append(env.Response.Files, createdFile)
 }
 
+// creates an IDReport from a base report
+func detailReport(incompatibilityReport *IncompatibilityReport) *IDReport {
```

Comment:
Is there a better name we can use instead of IDReport.  I'm not sure it provides enough context of what it is.

Reply:
IDReport -> FileDescriptiveReport